### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by EnricoMi**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
```
Comment (Debiday) [inline .github/ISSUE_TEMPLATE/bug_report.md:5]: Do you want to consider adding 'bug' to the labels?

**Comment by Debiday** [comment]
Hi @EnricoMi I think I understand where I structured it wrongly. I previously had 4 different PR Templates and one Issue Template. It's now a total of 3 Issue Templates, and 1 PR Template. It's probably clearer when the PR Template references an Issue rather than the other way around.

I have also updated the contributing guidelines to reflect this change in structure of PR and Issues and removed unnecessary templates and links. Hope this is working out better?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+3. Provide more context by answering these questions:
+
+* **Can you reliably reproduce the issue?** If not, provide details about how often the problem happens and under which conditions it normally happens.
+* If the problem is related to working with files (e.g. opening and editing files), **does the problem happen for all files and projects or only some?** Does the problem happen only when working with local or remote files (e.g. on network drives), with files of a specific type (e.g. only Scala or Python files), with large files or files in a specific encoding? Is there anything else special about the files you are using?
+
+4. Include details about your configuration and environment:
+
+* **What's the name and version of the OS you're using**?
+* **Are you running SDC in a virtual machine?** If so, which VM software are you using and which operating systems and versions are used for the host and the guest?
```
Comment (EnricoMi) [inline CONTRIBUTING.md:73]: not relevant

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
+
+**Describe alternatives you've considered**
+A clear and concise description of any alternative solutions or features you've considered.
+
+**Additional context**
+Add any other context or screenshots about the feature request here.
+
+(optional)
```
[the remark this answers is not in the document]
Comment (Debiday) [reply .github/ISSUE_TEMPLATE/feature_request.md:22]: Oops, I was referring to possible drawbacks(next point), but I see how it's misplaced. Removed it.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
```
[the remark this answers is not in the document]
Comment (Debiday) [reply .github/ISSUE_TEMPLATE/bug_report.md:28]: Ok, removed browser, added the others!

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
```
[the remark this answers is not in the document]
Comment (Debiday) [reply .github/ISSUE_TEMPLATE/bug_report.md:31]: Removed 👍

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+
+## I don't want to read this whole thing I just have a question!
+
+> **Note:** Please don't file an issue to ask a question.  You'll get faster results by using [Github Discussions](https://github.com/G-Research/spark-dgraph-connector/discussions), the primary SDC discussion forum.
+
+## What should I know before I get started?
+
+### The Spark Dgraph Connector
+
+The SDC is an open source project that supports databases in both Python and Scala. When considering a contribution to SDC, you might be unsure where to start or how to report a bug. This document should help you with that.
```
[the remark this answers is not in the document]
Comment (Debiday) [reply CONTRIBUTING.md:35]: Edited!

File: CONTRIBUTING.md (hunk 2 above)
Comment (Debiday) [reply CONTRIBUTING.md:73]: Removed 👍

**Comment by EnricoMi** [comment]
Approved!

**Comment by EnricoMi** [comment]
Look at that: 
grafik

**Comment by Debiday** [comment]
This looks much friendlier indeed 🎉 Thank you @EnricoMi for your suggestion!

**Pull request merged**